default-detekt-config.yml

```diff
@@ -0,0 +1,4740 @@
+
```

Comment:
Does this trigger any checks on github? Or is it an internal tool for intellij?

Reply:
It's a static code analysis tool integrated with gradle, so we can configure the CI to trigger a check on every PR.
[https://detekt.github.io/detekt/](https://detekt.github.io/detekt/)